acquia/cli
# CLI-685: Set up email:info command

**Motivation**
Supplements bulk Platform Email functionality/support by showing users the verification statuses of email domains registered to a subscription, as well as associated (or dissociated) domains for each application of the subscription.

**Proposed changes**
Add a new `email:info` command

**Alternatives considered**

**Testing steps**

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. `./bin/acli email:info`
4. Verify that CSV files are successfully written to a folder called "domain-info-*" in the directory where `email:info` is being run.

**Merge requirements**
